### PR TITLE
Added 3 cheatsheets 

### PR DESCRIPTION
**Pull request by morhadi**
## What does this PR do?
Add resources
Added 3 cheatsheets
* pyspark
* js 
* python

## For resources
this is the first pyspark cheatsheet here

### Why is this valuable (or not)?
valuable especially for pyspark

### How do we know it's really free?
they're websites (on github)

### For book lists, is it a book? For course lists, is it a course? etc.
They're GitHub repos

## Checklist: (done ✅)
- [ ✅] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ✅] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ✅] Include author(s) and platform where appropriate.
- [ ✅] Put lists in alphabetical order, correct spacing.
- [ ✅] Add needed indications (PDF, access notes, under construction).
- [ ✅] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: more/free-programming-cheatsheets.md [hunk 1]
```diff
@@ -350,6 +352,7 @@
 * [Numpy Pandas Cheat Sheet](https://www.kaggle.com/code/lavanyashukla01/pandas-numpy-python-cheatsheet) - Kaggle (HTML)
 * [Official Matplotlib cheat sheets](https://github.com/matplotlib/cheatsheets) - Matplotlib.org (LaTeX, PDF)
 * [Python 3 Cheat Sheet](https://perso.limsi.fr/pointal/_media/python:cours:mementopython3-english.pdf) - Laurent Pointal (PDF)
+* [Python Cheat Sheet](https://github.com/wilfredinni/python-cheatsheet) - Wilfredinni (HTML)
```
Comment (eshellman) [inline more/free-programming-cheatsheets.md:355]: should use https://www.pythoncheatsheet.org/

File: more/free-programming-cheatsheets.md [hunk 2]
```diff
@@ -386,6 +389,11 @@
 * [Pandas, Numpy, Python Cheatsheet](https://www.kaggle.com/code/lavanyashukla01/pandas-numpy-python-cheatsheet) - Kaggle (HTML)
 
 
+#### PySpark
+
+* [PySpark Cheat Sheet](https://github.com/kevinschaich/pyspark-cheatsheet) - Kevin Schaich (HTML)
```
Comment (eshellman) [inline more/free-programming-cheatsheets.md:394]: It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category. Just put it in the python category

**Comment by morhadi** [comment]
@eshellman Is it fine now?

File: more/free-programming-cheatsheets.md [hunk 3]
```diff
@@ -386,6 +389,12 @@
 * [Pandas, Numpy, Python Cheatsheet](https://www.kaggle.com/code/lavanyashukla01/pandas-numpy-python-cheatsheet) - Kaggle (HTML)
 
 
+#### PySpark
+
+* [PySpark Cheat Sheet: Spark in Python] (https://www.datacamp.com/cheat-sheet/pyspark-cheat-sheet-spark-in-python) - DataCamp (HTML)
```
Comment (eshellman) [inline more/free-programming-cheatsheets.md:394]:
```suggestion
* [PySpark Cheat Sheet: Spark in Python](https://www.datacamp.com/cheat-sheet/pyspark-cheat-sheet-spark-in-python) - DataCamp (HTML)
```

**Pull request merged**